fail to compile 6.1.0_ee5f303

**Description of the problem**
when running make:
libgmt.so.6.1.0: undefined reference to `support_rgb_to_hsv'


<!-- Please be as detailed as you can when describing an issue. The more information we have, the easier it will be for us to track this down. -->

**Full script that generated the error**

<!-- It helps a lot if you share any data files required to run your script. You can drag-and-drop them here or paste a link to Dropbox/GoogleDrive/etc. -->
Replacing support_rgb_to_hsv to gmt_rgb_to_hsv, as shown bellow, it appears to run correctly
```
--- a/src/gmt_support.c
+++ b/src/gmt_support.c
@@ -833,7 +833,7 @@ GMT_LOCAL bool support_gethsv (struct GMT_CTRL *GMT, char *line, double hsv[]) {
        if (count == 2) {       /* r/g/b */
                n = sscanf (buffer, "%d/%d/%d", &irgb[0], &irgb[1], &irgb[2]);
                if (n != 3 || support_check_irgb (irgb, rgb)) return (true);
-               support_rgb_to_hsv (rgb, hsv);
+               gmt_rgb_to_hsv (rgb, hsv);
                return (false);
        }```


**Full error message**

<!-- Tip: you'll get more detailed error messages if you add -Vd to your gmt commands. -->

```
PASTE ERROR MESSAGE HERE
```


**System information**

* Operating system:Ubuntu 18.04.2 LTS
* Version of GMT: 6.1.0_ee5f303

--- a/src/gmt_support.c
+++ b/src/gmt_support.c
@@ -833,7 +833,7 @@ GMT_LOCAL bool support_gethsv (struct GMT_CTRL *GMT, char *line, double hsv[]) {
 	if (count == 2) {	/* r/g/b */
 		n = sscanf (buffer, "%d/%d/%d", &irgb[0], &irgb[1], &irgb[2]);
 		if (n != 3 || support_check_irgb (irgb, rgb)) return (true);
-		support_rgb_to_hsv (rgb, hsv);
+		gmt_rgb_to_hsv (rgb, hsv);
 		return (false);
 	}
 

